api/controllers/OpenEAppController.js

```diff
@@ -96,7 +100,31 @@ const OpenEAppController = {
     },
 
     _formatDate(date) {
-        return moment(date).format('DD MMMM YYYY');
+        return dayjs(date).format('DD MMMM YYYY');
+    },
+
+    _calculateDaysLeftToDownload(req) {
+        const urlObj = new URL(
+            `${req.protocol}://${req.headers.host}${req.originalUrl}`
+        );
+        const queryParams = new URLSearchParams(urlObj.search);
+        if (!queryParams.has('completedDate')) {
+            throw new Error('No date value found in url');
+        }
+        const completedDate = queryParams.get('completedDate');
+        const currentDate = dayjs(Date.now());
+        const differenceBetweenCurrentAndCompletedDate = currentDate.diff(
+            completedDate,
+            'day'
+        );
+        const daysLeftToDownloadDoc =
+            MAX_DAYS_TO_LEGALISE -
```

Comment:
should it be `MAX_DAYS_TO_DOWNLOAD`? or `daysLeftToLegalise`? it's not clear that these two variables should be or are related since their names are different

Reply:
I admit it's a bit confusing. I'll fix that 👍